tests/js/spec/actionCreators/navigation.spec.jsx

```diff
@@ -21,6 +22,7 @@ describe('navigation ActionCreator', () => {
 
   afterEach(() => {
     ProjectsStore.reset();
```

Comment:
are the resets required if we're doing `teardown`?

Reply:
Yeah, teardown just clears the listeners, but some stores also include a "reset" which clears their internal state. We should consolidate the two I think, but that is for another time